configure.ac

```diff
@@ -770,6 +770,28 @@ VIPS_CFLAGS="$VIPS_CFLAGS $NIFTI_CFLAGS"
 VIPS_INCLUDES="$VIPS_INCLUDES $NIFTI_INCLUDES"
 VIPS_LIBS="$VIPS_LIBS $NIFTI_LIBS"
 
+# jpeg-xl
+AC_ARG_WITH([libjxl],
+  AS_HELP_STRING([--without-libjxl], 
+                 [build without libjxl (default: test)]))
+
+if test x"$with_libjxl" != x"no"; then
+  PKG_CHECK_MODULES(LIBJXL, libjxl_threads >= 0.3.7 libjxl >= 0.3.7,
```

Comment:
I had to install the `libbrotli-dev` package to get this to work, although I think this might be a bug in libjxl as it statically-links libbrotli but leaks the dependency in `Requires.private`.

Reply:
Yes, another paper cut around jxl :(